app/models/permissions/technician_permission.rb

```diff
@@ -5,10 +5,41 @@
 #
 module Permissions
   class TechnicianPermission < ScientistPermission
+    # - Technician's cannot edit Admin users or set users to admin type
+    # - Technician's cannot edit the protect flag within Locations
     def initialize(user)
       super
       allow :move_locations, [:create]
-      allow :empty_locations, [:create]
+      allow :location_types, [:create, :update]
+      allow :teams, [:create, :update]
+      allow :users, [:create, :update] do |record|
+        check_user(record)
+      end
+      allow :locations, [:create, :update] do |record|
+        check_locations(record)
+      end
+    end
+
+    private
+
+    def check_user(record)
```

Comment:
Again it might be a better idea to change this so it just returns a boolean. As long as we are telling the user they are not authorised that should be enough. It might also be worth only having the ability to create a Technician or Admnistrator.